.github/workflows/code_review.yaml

```diff
@@ -13,6 +13,6 @@ permissions:
 
 jobs:
   code_review:
-    uses: pagopa/dx/.github/workflows/js_code_review.yaml@main
+    uses: pagopa/dx/.github/workflows/js_code_review.yaml@19c975849e973ea0d3e43ed93c797a1b0addd02c
```

Comment:
I'd leave main

Reply:
I decided to keep the SHA at the end, to make the update explicit and not inadvertently break the workflow for developers.

To compensate for this choice, [I added a codemod dedicated to updating the js code review worklfow](https://github.com/pagopa/dx/pull/924)

In the future, we can introduce a `dx up` command that updates all the shas/workflows